docs/documentation/development/bindings/xml-reference.md

```diff
@@ -101,6 +101,19 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>criteria.name</td><td>The name of the context related filter.</td></tr>  
 </table>
 
+### Supported contexts
+
+Context is used to provide some semantic details about the parameter. The UI use it to render different kind of input widgets. Currently, PaperUI supports following contexts:
```

Comment:
Replace "Currently, PaperUI supports following contexts" by "The following contexts require a specific format of the content".